model/document.js

```diff
@@ -129,6 +129,25 @@ module.exports = memoize(function (db) {
 		isReleased: { type: db.Boolean, value: function () {
 			return this.status === 'approved';
 		} },
+		toWebServiceJSON: {
+			value: function (ignore) {
+				var data = {
+					code: this.uniqueKey,
+					files: [],
+					data: null
+				};
+				if (this.dataForm.constructor !== this.database.FormSectionBase) {
+					data.data = this.dataForm.toWebServiceJSON();
```

Comment:
Yes, not a big problem. There is a chance. We can have file type nested on any object. I will cut files from `dataForm` of documents only for now.